src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// Write code here
+import React from 'react';
+import './Person.scss';
+
+export const Person = ({
+  name, age, isMarried, sex, partnerName,
+}) => (
+
+  <section className="Person">
```

Comment:
```suggestion
}) => (
  <section className="Person">
```

Reply:
pay attention to the comments:
```suggestion
}) => (
  <section className="Person">
```